libraries/classes/User.php

```diff
@@ -133,4 +133,14 @@ public static function isLoggedIn()
       return false;
     }
   }
+  public static function sendFeedback(){
+    $db = new Factory;
+    $mysql = $db->getDBO();
+
+    $name = $_POST['name'];
```

Comment:
Instead of this get like this. https://github.com/NJACKWinterOfCode/iitpConnect/blob/492132f39dfee16e3f50adc84047591700062f0f/controllers/LoginController.php#L24